capa/features/extractors/vmray/extractor.py

```diff
@@ -72,8 +65,9 @@ def __init__(self, analysis: VMRayAnalysis):
         self.global_features = list(capa.features.extractors.vmray.global_.extract_features(self.analysis))
 
     def get_base_address(self) -> Address:
-        # value according to the PE header, the actual trace may use a different imagebase
-        return AbsoluteVirtualAddress(self.analysis.base_address)
+        # value according to submission file header, the actual trace may use a different imagebase
+        # value may be zero for certain submission file types, e.g. PS1
```

Comment:
should we use NO_ADDRESS for this?

Reply:
i honestly have no idea. it's ok to keep as is.

Reply:
I think this is a good change, see https://github.com/mandiant/capa/pull/2571/commits/ec1f11e54aa10a8fbb7aba18bbd21234178fa186